post_installation_configuration/cluster-tasks.adoc

```diff
@@ -322,6 +322,11 @@ include::modules/nodes-cluster-enabling-features-cli.adoc[leveloffset=+2]
 == etcd tasks
 Back up etcd, enable or disable etcd encryption, or defragment etcd data.
 
+[NOTE]
+====
+If you deployed a bare-metal cluster by using the Assisted Installer, as part of your post-installation tasks, you can scale the cluster up to 5 nodes. For more information, see xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#etcd-node-scaling_recommended-etcd-practices[Node scaling for etcd].
```

Comment:
Similarly to the previous note, I think the requirement is just that the clusters are bare metal, regardless of how they were installed, as the resulting clusters from the different installation methods are all the same.